CHANGELOG.md

```diff
@@ -1,3 +1,11 @@
+## v0.120.4, 1 October 2020
+
+- Go: Bump golang to v1.15.2 (thanks, @chenrui333!)
+- Bump phpstan/phpstan from 0.12.45 to 0.12.47 in /composer/helpers
+- Upgrade Python to 3.8.6 (@ulgens)
+- Handle empty pipfile requirement string
+- Teach FileFetcher to fetch from disk if local repository clone is present
```

Comment:
```suggestion
- Teach FileFetcher to fetch from disk if local repository clone is present
- Bundler: refactor DependencySource from LatestVersionFinder
```

Wording 🤷. Related to https://github.com/dependabot/dependabot-core/pull/2589/files#r498435945 , but suggestions there affect lines 3-7